Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridCellsPresenter.cs

```diff
@@ -246,11 +246,11 @@ protected override Size MeasureOverride(Size availableSize)
                     {
                         cell.Measure(new Size(column.ActualMaxWidth, measureHeight));
                         this.OwningGrid.AutoSizeColumn(column, cell.DesiredSize.Width);
-                        column.ComputeLayoutRoundedWidth(totalDisplayWidth);
+                        column.ComputeLayoutRoundedWidth(totalDisplayWidth, GetScale());
```

Comment:
>GetScale [](start = 76, length = 8)

Here too ComputeLayoutRoundedWidth could use OwningGrid internally to calculate the scale, doesn't need it as a parameter.

Reply:
Addressed by https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3206/commits/74b7c128681b06c5bc65220eeea1716dacb6f6c5